TypeCobol/AntlrGrammar/CobolCodeElements.g4

```diff
@@ -8260,18 +8260,27 @@ execStatementText: ExecStatementText;
 execStatementEnd: END_EXEC;
 
 //FOR SQL
+//TODO 
```

Comment:
No, I'd like to have a more explicit TODO on single_table_or_view_reference, something like '//TODO Add period-specification syntax'. A single '//TODO' here does not help understand what is missing.